WordPress/Classes/ViewRelated/Site Creation/Shared/SiteCreationAnalyticsHelper.swift

```diff
@@ -147,6 +147,12 @@ class SiteCreationAnalyticsHelper {
             if let previewMode = property as? PreviewDevice {
                 result.merge([previewModeKey: previewMode.rawValue]) { (_, new) in new }
             }
+
+            if let dictionary = property as? [String: Any] {
```

Comment:
Totally optional, just wanted to share my thoughts on this:

It looks like `commonProperties` was created to handle specific, yet common properties used with multiple events.  I'd consider adding the recommended property in `trackSiteDesignSelected` to keep this function focused since we only have one call site that needs it. Though not likely, it would also help reduce the risk of a duplicate key situation.

Reply:
The reason for adding this code here is to add more flexibility to this analytics helper. Sure we can add a single property and pass it, but then if we need to add one more, we will need to add yet another specialized property and call. This way (unless I'm not seeing something), you have the freedom of adding properties that are not necessarily tied to specific types, and could be "common" as well. What do you think?

Reply:
Yep, that's fine, just wanted to share my thoughts. 👍